server/src/main/java/org/apache/iotdb/db/metadata/idtable/IDTable.java

```diff
@@ -129,7 +141,7 @@ void updateLastCache(
    * @param deviceName device name of the time series
    * @return device entry of the timeseries
    */
-  public DeviceEntry getDeviceEntry(String deviceName);
+  DeviceEntry getDeviceEntry(String deviceName);
```

Comment:
Why make this as ```default```?

Reply:
The default methods of java interfaces are public and abstract, so there is no need to add public.